<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LumWindow</class>
 <widget class="QMainWindow" name="LumWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Envio - Thermostat</string>
  </property>
  <property name="styleSheet">
   <string notr="true">Background: #232123;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>301</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayoutBtn1">
       <item>
        <widget class="QPushButton" name="AccueilBtn">
         <property name="font">
          <font>
           <family>Segoe UI,sans-serif</family>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
font-family:&quot;Segoe UI&quot;, sans-serif;
	text-transform: uppercase;
	outline: 0;
	background: #5cb85c;
	border: 0;
	padding: 7px;
	color: #FFFFFF;
	font-size: 14px;
	border-radius: 7px;
}

QPushButton:hover
{
font-family:&quot;Segoe UI&quot;, sans-serif;
	text-transform: uppercase;
	outline: 0;
	background: #449d44;
	border: 0;
	padding: 7px;
	color: #FFFFFF;
	font-size: 14px;
	border-radius: 7px;
}</string>
         </property>
         <property name="text">
          <string>Accueil</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="TempEditBtn">
         <property name="font">
          <font>
           <family>Segoe UI,sans-serif</family>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
font-family:&quot;Segoe UI&quot;, sans-serif;
	text-transform: uppercase;
	outline: 0;
	background: #d9534f;
	width: 100%;
	border: 0;
	padding: 7px;
	color: #FFFFFF;
	font-size: 14px;
	border-radius: 7px;
}

QPushButton:hover
{
font-family:&quot;Segoe UI&quot;, sans-serif;
	text-transform: uppercase;
	outline: 0;
	background: #c9302c;
	width: 100%;
	border: 0;
	padding: 7px;
	color: #FFFFFF;
	font-size: 14px;
	border-radius: 7px;
}</string>
         </property>
         <property name="text">
          <string>Température</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="OpacEditBtn">
         <property name="text">
          <string>Vitres</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="LumFixLabel">
       <property name="font">
        <font>
         <family>Segoe UI,sans-serif</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font-family:&quot;Segoe UI&quot;, sans-serif;
 text-transform: uppercase;
 outline: 0; 
width: 100%; 
border: 0;
 padding: 10px;
color: #AAA;
font-size: 14px;</string>
       </property>
       <property name="text">
        <string>Lumière :</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSlider" name="LumHorizontalSlider">
       <property name="styleSheet">
        <string notr="true">QSlider::groove:horizontal {
border: 1px solid #bbb;
background: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 0,
    stop: 0 #003737, stop: 1 #00cccc);
height: 10px;
border-radius: 4px;
}
/*
QSlider::sub-page:horizontal {
background: qlineargradient(x1: 0, y1: 0,    x2: 0, y2: 1,
    stop: 0 #00CCCC, stop: 1 #008C8C);
background: qlineargradient(x1: 0, y1: 0.2, x2: 1, y2: 1,
    stop: 0 #00CCCC, stop: 1 #008C8C);
border: 1px solid #777;
height: 10px;
border-radius: 4px;
}
*/
QSlider::add-page:horizontal {
background: transparent;
border: 1px solid #777;
height: 10px;
border-radius: 4px;
}

QSlider::handle:horizontal {
background: #1D797B;
border: 1px solid #777;
width: 13px;
margin-top: -2px;
margin-bottom: -2px;
border-radius: 4px;
}

QSlider::handle:horizontal:hover {
background: #1D797B;
border: 1px solid #444;
border-radius: 4px;
}

QSlider::sub-page:horizontal:disabled {
background: #bbb;
border-color: #999;
}

QSlider::add-page:horizontal:disabled {
background: #eee;
border-color: #999;
}

QSlider::handle:horizontal:disabled {
background: #eee;
border: 1px solid #aaa;
border-radius: 4px;
}</string>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>70</number>
       </property>
       <property name="sliderPosition">
        <number>70</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayoutBtn2">
       <item>
        <widget class="QPushButton" name="PlanningEditBtn">
         <property name="font">
          <font>
           <family>Segoe UI,sans-serif</family>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
font-family:&quot;Segoe UI&quot;, sans-serif;
	text-transform: uppercase;
	outline: 0;
	background: #5cb85c;
	border: 0;
	padding: 7px;
	color: #FFFFFF;
	font-size: 14px;
	border-radius: 7px;
}

QPushButton:hover
{
font-family:&quot;Segoe UI&quot;, sans-serif;
	text-transform: uppercase;
	outline: 0;
	background: #449d44;
	border: 0;
	padding: 7px;
	color: #FFFFFF;
	font-size: 14px;
	border-radius: 7px;
}</string>
         </property>
         <property name="text">
          <string>Planning</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ConfigEditBtn">
         <property name="font">
          <font>
           <family>Segoe UI,sans-serif</family>
           <pointsize>-1</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{
font-family:&quot;Segoe UI&quot;, sans-serif;
	text-transform: uppercase;
	outline: 0;
	background: #5cb85c;
	border: 0;
	padding: 7px;
	color: #FFFFFF;
	font-size: 14px;
	border-radius: 7px;
}

QPushButton:hover
{
font-family:&quot;Segoe UI&quot;, sans-serif;
	text-transform: uppercase;
	outline: 0;
	background: #449d44;
	border: 0;
	padding: 7px;
	color: #FFFFFF;
	font-size: 14px;
	border-radius: 7px;
}</string>
         </property>
         <property name="text">
          <string>Configuration</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="LumLabel">
       <property name="font">
        <font>
         <family>Segoe UI,sans-serif</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font-family:&quot;Segoe UI&quot;, sans-serif;
text-transform: uppercase;
outline: 0;
width: 100%;
border: 0;
padding: 10px;
color: #0288d1;
font-size: 46px;
margin: 0, -5, 0,0;</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
